examples/mlperf/model_train.py

```diff
@@ -346,8 +346,222 @@ def train_retinanet():
   pass
 
 def train_unet3d():
-  # TODO: Unet3d
-  pass
+  """
+  Trains the UNet3D model.
+
+  Instructions:
+  1) Run the following script from the root folder of `tinygrad`:
+  ```./examples/mlperf/scripts/setup_kits19_dataset.sh```
```

Comment:
this failed because `BASEDIR` is not set

even better is to have a default for `BASEDIR`

Reply:
that's a good call. i was initially contemplating about this when i was modifying the dataset shell script.

note that i also updated the doc string slightly to show the availability of `BASEDIR` when running the shell script.